src/Expose/FilterCollection.php

```diff
@@ -88,6 +92,21 @@ public function setFilterData($data)
     }
 
     /**
+     * Alter the impact level of a specific filter id.
+     *
+     * @param integer $filterId
+     * @param integer $impact
+     */
+    public function setFilterImpact($filterId, $impact) {
+        $filter = $this->getFilterData($filterId);
+
+        if($filter === null) {
+            return;
```

Comment:
@quantumpacket you confused me. Feel free to submit a patch for that once I recommit @mpedrummer's code.